### PR TITLE
Allow editing of Paragraphs via quick links

### PR DESCRIPTION
**Pull request by amitaibu**
- Using [Paragraphs edit](https://www.drupal.org/project/paragraphs_edit) module
- Requires a patch to PEVB - https://git.drupalcode.org/project/pluggable_entity_view_builder/-/merge_requests/23

| View | Edit Paragraph |
| -- | -- |
| image | image |

### CODE REVIEW

**Comment by amitaibu** [comment]
cc @mariano-dagostino

**Review by mariano-dagostino — APPROVED** [review]
I tried this and works fine. Also in preview mode :100: 

@amitaibu Just wonder if makes sense to add this to PEVB and conditionally check if `paragraph_edit` module exists.

**Comment by amitaibu** [comment]
>  Just wonder if makes sense to add this to PEVB and conditionally check if paragraph_edit module exists.

That's a good idea

**Comment by amitaibu** [comment]
@mariano-dagostino can you please review https://git.drupalcode.org/project/pluggable_entity_view_builder/-/merge_requests/24

**Comment by amitaibu** [comment]
@mariano-dagostino ^^

**Comment by mariano-dagostino** [comment]
@amitaibu https://git.drupalcode.org/project/pluggable_entity_view_builder/-/merge_requests/24/diffs

**Comment by mariano-dagostino** [comment]
@amitaibu Sorry I forgot to save the review: https://git.drupalcode.org/project/pluggable_entity_view_builder/-/merge_requests/24/diffs#note_155044

File: composer.lock [hunk 1]
```diff
@@ -3174,17 +3223,17 @@
...
-                "reference": "1.1.13"
+                "reference": "1.1.15"
             },
             "dist": {
                 "type": "zip",
-                "url": "https://ftp.drupal.org/files/projects/pluggable_entity_view_builder-1.1.13.zip",
-                "reference": "1.1.13",
-                "shasum": "3af796f9ee1f33f575829c7c92da14a80585e5d8"
+                "url": "https://ftp.drupal.org/files/projects/pluggable_entity_view_builder-1.1.15.zip",
+                "reference": "1.1.15",
```
Comment (amitaibu) [inline composer.lock:3235]: Upgraded to 1.1.15

**Pull request merged**